lib/request.ts

```diff
@@ -2,10 +2,11 @@ import {Message, MessagePayload} from "./messageFactory";
 import {Requester} from "./requester";
 import validateWithSchema = require("./validateWithSchema");
 
-const _ = require("lodash");
+const clone = require("lodash.clone");
```

Comment:
@Kumjami please read the (clone documentation)[The own enumerable properties of `arguments` objects are cloned]. It behaves the same as `Object.assign`